### PR TITLE
[lpg_planner] install lpg_planner from jsk-ros-pkg/archieve

### PR DESCRIPTION
**Pull request by knorth55**
we need to manually install `lpg` into `~/Downloads` for current lpg_planner.
this PR change the installation and automatically install `lpg` in `catkin build`.

### CODE REVIEW

File: 3rdparty/lpg_planner/Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+all: installed
+
+INSTALL_DIR=`rospack find lpg_planner`
```
Comment (708yamaguchi) [inline 3rdparty/lpg_planner/Makefile:3]: `INSTALL_DIR` is used ?

File: 3rdparty/lpg_planner/Makefile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+FILENAME = lpg1.2-linux.tar.gz
+TARBALL = build/$(FILENAME)
+TARBALL_URL = http://github.com/jsk-ros-pkg/archives/raw/master/$(FILENAME)
```
Comment (708yamaguchi) [inline 3rdparty/lpg_planner/Makefile:6]: Compared to https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/3rdparty/julius/Makefile.grammar-kit ,
the software version should be fixed.

**Review by 708yamaguchi — COMMENTED** [review]
I checked that LPG-1.2-linux is installed (uninstalled) under build dir by catkin build (clean).

After installing, I can do
roslaunch pddl_planner example.launch
https://github.com/jsk-ros-pkg/jsk_planning/tree/master/pddl/pddl_planner/demos/2016_kamada_durative

I added 2 comments about this PR.

File: 3rdparty/lpg_planner/Makefile (hunk 1 above)
Comment (knorth55) [reply 3rdparty/lpg_planner/Makefile]: thanks, i removed.

File: 3rdparty/lpg_planner/Makefile (hunk 2 above)
Comment (knorth55) [reply 3rdparty/lpg_planner/Makefile:6]: ok, I add version

**Review by 708yamaguchi — APPROVED** [review]

**Pull request merged**